packages/common/src/chains/mainnet.json

```diff
@@ -82,6 +82,11 @@
       "block": 13773000,
       "forkHash": "0x20c327fc"
     },
+    {
+      "name": "grayGlacier",
+      "block": 15050000,
+      "forkHash": null
```

Comment:
Please calculate an explicit forkHash here, you can do this with the respective method in the `Common` library. If you would get some confirmation that the forkHash is correct - e.g. by looking at other client implementation GrayGlacier HF PRs - that would also be good.